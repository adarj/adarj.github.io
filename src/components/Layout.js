import React from "react"
import SEO from "./SEO"

const Layout = () => (
  <>
    <SEO />
  </>
)

export default Layout
