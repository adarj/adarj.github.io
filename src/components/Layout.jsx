import React from 'react';
import SEO from './SEO';
import GlobalStyle from '../styles/global';

const Layout = (props) => (
  <>
    <SEO title={props.title} />
    <GlobalStyle />
  </>
);

export default Layout;
