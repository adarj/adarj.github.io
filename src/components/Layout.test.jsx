import React from 'react';
import { shallow } from 'enzyme';

import Layout from './Layout';

describe('Layout', () => {
  it('renders correctly', () => {
    const tree = shallow(<Layout title="John Doe" />)
    expect(tree).toMatchSnapshot()
  });
});
