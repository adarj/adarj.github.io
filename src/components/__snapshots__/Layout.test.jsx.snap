// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<Fragment>
  <SEO
    title="John Doe"
  />
  <GlobalStyleComponent />
</Fragment>
`;
